.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 Open Networking Foundation
+
+name: Build PFCPsim
+on: [push, pull_request]
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.16
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
+
+      - name: Check out repository
+        uses: actions/checkout@v1
+
+      - name: Build
+        run: go build cmd/pfcpsim-client/main.go
```

Comment:
shouldn't we use docker build?

Reply:
yes. Now it uses docker build.
CI however does not release during test step. Need to investigate